client/src/document/index.scss

```diff
@@ -3,15 +3,29 @@
 @import "~@mdn/minimalist/sass/vars/borders";
 @import "~@mdn/minimalist/sass/vars/typography";
 
+@import "~@mdn/minimalist/sass/mixins/animation";
 @import "~@mdn/minimalist/sass/mixins/utils";
 
 @import "./prism-overrides";
 
-.loading-document-placeholder article {
-  // Any huge margin bottom to keep the footer away when loading
-  margin-bottom: 1200px;
+/* we already disable all animation and transitions globally via a reset so */
```

Comment:
I don't understand this comment. Does it make sense to you ...in 6months time?

Reply:
Yup 😄 it's the reset used in minimalist. I can totally add that though, for additional context.